app/src/main/java/uk/ac/ebi/atlas/experimentpage/metadata/CellMetadataDao.java

```diff
@@ -166,7 +168,26 @@ public CellMetadataDao(SolrCloudCollectionProxyFactory solrCloudCollectionProxyF
                             // shouldn't be. Apparently we don't expect any cell ID to have more than one factor
                             // value. This was confirmed by curators in this Slack conversation:
                             // https://ebi-fg.slack.com/archives/C800ZEPPS/p1529592962001046
-                            tuple -> tuple.getStrings(METADATA_VALUE_TARGET_FIELD_NAME).get(0)));        }
+                            // Missing values of a given metadata type is technically an invalid condensed SDRF file,
+                            // but it’s a situation that we’ve encountered in the past and we want the web app to be
+                            // somewhat lenient about it and show the experiment, rather than show a cryptic 400 error;
+                            // instead we log an error and fill those fields with “Not available”.
+                            // Please see:
+                            // https://ebi-fg.slack.com/archives/C9P2QETJ9/p1624354966014500
+                            // https://ebi-fg.slack.com/archives/C800ZEPPS/p1624362599003900
+                            // https://www.pivotaltracker.com/story/show/177533187
+                            tuple -> {
+                                if (tuple.getStrings(METADATA_VALUE_TARGET_FIELD_NAME) == null ||
+                                    !tuple.getStrings(METADATA_VALUE_TARGET_FIELD_NAME).isEmpty()) {
```

Comment:
Wouldn't this condition result in the correct Cell Ids being logged as errors? Say for example, `cell_id` is `not null` and its name is `xyz` then this `if` condition would evaluate to true. As it has a `!` before `isEmpty()`.

Or am I missing something here?